### ISSUE TITLE
`black latest` is not working

### ISSUE THREAD

**Issue by amrox** [amrox/asdf-pyapp#2]
**Describe the bug**

**Steps to reproduce**

- Install this plugin for `black`
- note that `latest` version of `black` appears to be whitespace

```
❯ asdf install black latest
black  is already installed
```

**Comment by amrox** [comment]
Did some more digging. Currently all of `black`'s versions are:

```
18.3a0
18.3a1
18.3a2
18.3a3
18.3a4
18.4a0
18.4a1
18.4a2
18.4a3
18.4a4
18.5b0
18.5b1
18.6b0
18.6b1
18.6b2
18.6b3
18.6b4
18.9b0
19.3b0
19.10b0
20.8b0
20.8b1
21.4b0
21.4b1
21.4b2
21.5b0
21.5b1
21.5b2
21.6b0
21.7b0
21.8b0
21.9b0
21.10b0
```

There isn't one without a `a` or `b` suffix. It looks like they're all being filtered out by [this](https://github.com/asdf-vm/asdf/blob/4321980c3385ac1bafd77503c8ec77b26042d05b/lib/commands/command-latest.bash#L27).

One option appears to be adding a `latest-stable` script to our plugin. However, I'm not sure this is the best option since I'm not sure we want to override version handling for _all_ python apps. In turn, I'm not sure we want to put black-specific workarounds in our code either.

**Comment by amrox** [comment]
black released 21.1.0, so latest is now working! 🎉
